actions/techdocs-rewrite-relative-links/action.yaml

```diff
@@ -0,0 +1,72 @@
+name: "Techdocs: Rewrite relative links"
+description: Rewrite links inside techdocs pointing to outside resources
+
+inputs:
+  default-branch:
+    type: string
+    required: true
+  repo-url:
+    type: string
+    required: true
+  working-directory:
+    type: string
+    required: true
+  dry-run:
+    description: |
+      Execute link rewriting without updating the underlying files
+    type: boolean
+    required: false
+    default: false
+  checkout-path:
+    type: string
+    default: _action
+    required: false
+  skip-checkout:
+    description: |
+      If the action is already present in the workspace you can skip another checkout.
+    type: boolean
+    required: false
+    default: false
```

Comment:
suggestion: Could be (probably is) just me, but I find inverted options hard to think about - I'm unsure whether to use `true` or `false`. Suggest calling this `checkout` instead and then it'd look like `checkout: false` to skip, which is nice enough I think. :slightly_smiling_face:

Reply:
Done